Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -97,25 +97,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
     <string name="update_later">Later</string>
     <string name="update_none">Already on the latest %1$s version.</string>
-    <string name="update_check_failed">Failed to check for updates.</string>
+    <string name="update_check_failed">Could not fetch updates.</string>
```

Comment:
``Fetch`` implies that an update existed and it couldn't be got. That's not what this string is for.

Reply:
In my line of thinking you by that mean "upgrades".